x-pack/plugins/code/public/style/_mixins.scss

```diff
@@ -0,0 +1,23 @@
+@mixin euiScrollBar {
```

Comment:
What's the need for replicating the mixin?

Reply:
@cchaos It wasn't included in the imported mixins... Maybe we're doing something wrong though
```
@import 'src/legacy/ui/public/styles/_styling_constants';
```

Reply:
Hmm, it should be available because that file imports the mixins index file. https://github.com/elastic/kibana/blob/68a57affaa05b4d86250b60158f0517f41dd58f1/src/legacy/ui/public/styles/_styling_constants.scss#L9

What would happen if you removed your mixin and just left the `@include euiScrollBar;`?

Reply:
That's where I started, but I was getting a SCSS compilation error. I'm guessing it was more likely a spelling error cause it's working now.